stable/traefik/README.md

```diff
@@ -123,7 +123,14 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `accessLogs.enabled`            | Whether to enable Traefik's access logs                              | `false`                                   |
 | `accessLogs.filePath`           | The path to the log file. Logs to stdout if omitted                  | None                                      |
 | `accessLogs.format`             | What format the log entries should be in. Either `common` or `json`  | `common`                                  |
-
+| `metrics.prometheus.enabled`    | Enable/ Disable the `/metrics` endpoint for metric collection by Prometheus. | `false`                           |
```

Comment:
How about `Whether to enable` instead of `Enable/ Disable` to match other descriptions?

(applies to a few lines below too)